api/user.go

```diff
@@ -664,13 +684,15 @@ func loginLdap(c *Context, w http.ResponseWriter, r *http.Request) {
 func revokeSession(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
@crspeller This is the existing function and you can revoke any session, not just the current one.